Revert this commit because linking it's returning an undefined reference to "aligned_alloc":
	https://github.com/gcc-mirror/gcc/commit/d7b41a73a78cbd6f291bf8d4090638057b964c11

--- a/libstdc++-v3/config/abi/pre/gnu.ver
+++ b/libstdc++-v3/config/abi/pre/gnu.ver
@@ -2198,16 +2198,16 @@
     _ZNSt15__exception_ptr13exception_ptrC1EPv;
 
     # C++17 aligned new/delete
-    _Znw[jmy]St11align_val_t;
-    _Znw[jmy]St11align_val_tRKSt9nothrow_t;
-    _Zna[jmy]St11align_val_t;
-    _Zna[jmy]St11align_val_tRKSt9nothrow_t;
+    _ZnwmSt11align_val_t;
+    _ZnwmSt11align_val_tRKSt9nothrow_t;
+    _ZnamSt11align_val_t;
+    _ZnamSt11align_val_tRKSt9nothrow_t;
     _ZdlPvSt11align_val_t;
     _ZdlPvSt11align_val_tRKSt9nothrow_t;
-    _ZdlPv[jmy]St11align_val_t;
+    _ZdlPvmSt11align_val_t;
     _ZdaPvSt11align_val_t;
     _ZdaPvSt11align_val_tRKSt9nothrow_t;
-    _ZdaPv[jmy]St11align_val_t;
+    _ZdaPvmSt11align_val_t;
 
 } CXXABI_1.3.10;
 
